doc/source/stack_and_queue.rst

```diff
@@ -20,3 +20,5 @@ Usage of Queue::
   x = front(q)
   x = back(q)
   x = dequeue!(q)
+
+Both ``Stack`` and ``Queue`` implement the Iterator interface; iterating over ``Stack`` returns items in FILO order and iterating over ``Queue`` returns items in FIFO order. There is also a ``reverse_iter`` function implemented for both which returns items in the reverse order for each type (i.e. FIFO for ``Stack`` and FILO for ``Queue``).
```

Comment:
That's my fault! I started mix matching a few things yesterday. A good sign that one needs more sleep.